doc/modules/language-guide/pages/errors.adoc

```diff
@@ -11,7 +11,7 @@
 == Our Example API ==
 Let's assume we're building an API for a Todo application and want to expose a function that lets a user mark one of their Todo's as *Done*.
 To keep it simple we'll accept a `TodoId` and return an `Int` that represents how many seconds the Todo has been open.
-We're also assuming we're running in our own canister so we return an async value.
+We're also assuming we're running in our own actor so we return an async value.
```

Comment:
Usage of "actor" is a bit inconsistent. Above we say "actor, as an IC canister". and "canister smart contract". Here we just say "actor". Maybe we should decide on what a Motoko program running on IC is and consistently use that?

Reply:
I'll stick to actor once in Motoko land